frameworks/PHP/workerman/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+fw_depends mysql
```

Comment:
Please install php using the setup scripts we provide. this can be:
`fw_depends mysql php7 composer`

Reply:
it is not possible. "fw_depends php" does not have PCNTL capability.